### PR TITLE
Build tweaks

### PR DESCRIPTION
**Pull request by chris3torek**
This is a series of tweaks to build.py and container_build.py aimed at some usability improvement for development with DPDK updates if and when this is required, and for building with automated build systems (e.g., travis, where we've seen things like travis timeouts—we could work around these by tweaking the travis settings but I like this idea better).

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -317,41 +349,44 @@ def update_benchmark_path(path):
...
-    elif args.action == 'clean':
-        do_clean()
-    elif args.action == 'dist_clean':
-        do_dist_clean()
-    elif args.action == 'help':
-        print_usage(parser)
-    else:
-        print('Error - unknown command "%s".' % sys.argv[1], file=sys.stderr)
-        print_usage(parser)
+    cmds[args.action]()
```
Comment (changlan) [inline build.py]: 👍

**Comment by sirspinach** [comment]
Could you change .travis.yml to build in verbose by default?

**Comment by chris3torek** [comment]
I thought I'd leave that to a separate thing (mostly because I'm not sure which builds are timing out in the first place). Which one(s) are?

**Comment by chris3torek** [comment]
(This was discussed elsewhere but I'll add it here since I guess these discussions get archived on GitHub.) It looks like the place to tweak in travis is in another repo that just uses this one.

**Comment by changlan** [comment]
Merged, thanks!
image

**Pull request merged**